packages/client/src/components/interface/DownloadButton.tsx

```diff
@@ -252,19 +250,20 @@ function DownloadButtonComponent(props: DownloadButtonProps & HOCProps) {
     downloadConfigs.declarationStatus &&
     ['VALIDATED', 'CORRECTION_REQUESTED'].includes(
       downloadConfigs.declarationStatus
-    ) &&
-    userRole === ROLE_REGISTRATION_AGENT
+    )
+  // && userRole === ROLE_REGISTRATION_AGENT
 
+  // @TODO: Ask JPF how this should be handled?
   // field agents can only retrieve declarations
-  const isNotFieldAgent = !FIELD_AGENT_ROLES.includes(String(userRole))
+  //const isNotFieldAgent = !FIELD_AGENT_ROLES.includes(String(userRole))
```

Comment:
@jpye-finch In DownloadButton, there is a condition that only field agents can retrieve declarations. There is a check:
```
const isRetrieveableDeclarationsOfRegAgent =
    downloadConfigs.declarationStatus &&
    ['VALIDATED', 'CORRECTION_REQUESTED'].includes(
      downloadConfigs.declarationStatus
    ) &&
    userRole === ROLE_REGISTRATION_AGENT
    )
    
    const isNotFieldAgent = !FIELD_AGENT_ROLES.includes(String(userRole))
 ```
 
 then
 ```
     
        !isRetrieveableDeclarationsOfRegAgent &&
        isNotFieldAgent
  ```
  
  this opens up an assignment modal if the condition matches. How would this be handled with scopes?

Reply:
We will eventually remove this functionality. 

There is a proposed design to move Audit into the View record. So that you won't need to assign to the record history. 

Therefore a Field Agent type role won't need to assign to view the history. Which is currently the case.